packages/mco/components/disaster-recovery/drpolicy-list-page/drpolicy-list-page.tsx

```diff
@@ -167,11 +167,13 @@ export const DRPolicyListPage: React.FC = () => {
         isOpen={isModalOpen}
       />
       {drPolicies?.length === 0 ? (
+        // All length 0 cases are handled by EmptyPage
         <EmptyPage
           title={t('No disaster recovery policies yet')}
           buttonText={t('Create DRPolicy')}
           canAccess={!drPoliciesLoadError && drPoliciesLoaded}
           t={t}
+          isLoaded={drPoliciesLoaded || !!drPoliciesLoadError}
```

Comment:
don't u mean this ?? or in case of any error u are assuming it to be as loaded ??
```suggestion
          isLoaded={drPoliciesLoaded && !drPoliciesLoadError}
```

Reply:
`in case of any error u are assuming it to be as loaded ??` >> if so, then what will happen in case of any error, what will appear on screen ??

Reply:
> drPoliciesLoadError
Yes, I think both conditions are similar, But I will change to drPoliciesLoaded && !drPoliciesLoadError.
In case of an error, Just like ACM pages, The Empty page will disable the create DRPolicy button and display a default permission denied message over the button.

Reply:
I tested, drPoliciesLoaded && !drPoliciesLoadError won't work when any user comes with DRPolicy read permission, it leads to keep loading. In that case, drPoliciesLoaded is always false and drPoliciesLoadError has permission-related errors.

Reply:
ack... can u plz add a comment for the reason of doing this...

Reply:
done